README.md

```diff
@@ -4,7 +4,8 @@
 <img src="./src/assets/logo.svg" alt="Logo" width="150px">
   <h2><b>Crypto Angels</b></h2>
 </div>
-   <h4><b>Readme</b></h4>
+
+<h4 align="center"><b>Readme</b></h4>
```

Comment:
**I can see you have done a great job.** 

- [ ] However, there are some places that need to be fixed up. Let's get started.
In this project `eslint-disable..` is not allowed. Kindly remove `all` `eslint-disabler` from your project. I was not able to provide all the screenshots but this is an example of them:

<img width="729" alt="Screenshot 2023-05-25 at 19 20 45" src="https://github.com/Metaverse-Nabeel/Stocks-Metrics/assets/69011963/85caf7a7-5446-4479-a7dc-27dca919a03f">

/
> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.

Reply:
Hello @Rustamxon7 

I have removed all the ESlint-disable lines from the code except for the `prop-types`. I have tried many different solutions but none worked for that.
If you have some solution please let me know. 

Thank you

Reply:
- [ ] How about disabling it in the .eslintrc.json?

Reply:
I did that, and it works for all other ESlint-disables but not prop-types. I tried installing it and adding it to the `package.json` dependencies but it's not working and gives an error.

Reply:
In this case, you can submit your code again. If the issue was not able to fix you are free to continue.

Reply:
Thank you

Reply:
Yes, you can :)